CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+-   Dynamic table handler to reload the data from an external source.
+-   Added new story to reload data with sample code.
+-   Implemented className and style properties to the table and the actions.
+-   Added reload tests.
+-   Changed max-lines allowed in DataTableInterfaces.ts.
```

Comment:
This really is a detail but this should go under a `Changed` section.

ref: https://keepachangelog.com/en/1.0.0/

Reply:
Moved changes to the correct section